packages/runtime/src/lib/constants.ts

```diff
@@ -29,6 +29,11 @@ export const MaxAppProgramLen = 1024;
 export const MAX_ALGORAND_ACCOUNT_ASSETS = 1000;
 export const MAX_ALGORAND_ACCOUNT_APPS = 10;
 
+// for byteslice arithmetic ops, inputs are limited to 64 bytes,
+// but ouput can be upto 128 bytes (eg. when using b+ OR b*)
+export const MAX_INPUT_BYTE_LEN = 64;
+export const MAX_OUTPUT_BYTE_LEN = 128;
```

Comment:
I think we should rename BYTE -> BIT. Shall we have `MAX_BIT_LEN` and `MAX_BIT_PLUS_LEN`? It seams that some operations for output will also return 64 bits.

Reply:
`64` & `128` represent bytes here (also each value in a `uint8array` represents 1byte/8 bits). So changing to bit would look like:
```js
export const MAX_BIT_LEN = 64 * 8;
export const MAX_BIT_PLUS_LEN = 128 * 8;
```
Let me know if i should still change it

Reply:
but this is used for numbers, right? I don't think Algorand support `64*8` bit number. This is huge and more then anyone could imagine.

Reply:
This is used for byte slice arithmatic. This is not natively on numbers. 
For eg.
```
byte "A"
byte "B"
b+ // "A" + "B" 
```
So we convert "A", "B" => uint (using `bigEndianBytesToBigInt`), then add them, and convert result back to byte. 
For bytes, the max limit is 64 (for input) & 128 (for output). But it can be handled by bigint (yes these would be veryy large numbers). I have added tests for it.

Reply:
let's put this link next to the limits above: https://github.com/algorand/go-algorand/blob/bd5a00092c8a63dba8314b97851e46ff247cf7c1/data/transactions/logic/eval.go#L1302

Reply:
ok